### PR TITLE
MWPW-136802: CaaS Configurator UI controls update

### PR DESCRIPTION
**Pull request by cmiqueo**
This PR adds some minor css tweaks to the CaaS Configurator tool.
It also resolves the issue reported on Jira ticket  MWPW-131760 where the max-height of the accordion is limiting the number of elements an author can add.

Resolves: [MWPW-131760](https://jira.corp.adobe.com/browse/MWPW-131760) & [MWPW-136802](https://jira.corp.adobe.com/browse/MWPW-136802)

**Test URLs:**
- Before: [Content as a Service - Thursday, August 3, 2023 at 15:24 (no-lazy)](https://milo.adobe.com/tools/caas#eyJhbmFseXRpY3NUcmFja0ltcHJlc3Npb24iOmZhbHNlLCJhbmFseXRpY3NDb2xsZWN0aW9uTmFtZSI6IiIsImFuZExvZ2ljVGFncyI6W3siaW50cmFUYWdMb2dpYyI6Ik9SIiwiYW5kVGFncyI6WyJjYWFzOmNvbnRlbnQtdHlwZS9yZXBvcnQiLCJjYWFzOmNvbnRlbnQtdHlwZS92aWRlbyIsImNhYXM6Y29udGVudC10eXBlL2FydGljbGUiLCJjYWFzOmNvbnRlbnQtdHlwZS93ZWJpbmFyIiwiY2Fhczpjb250ZW50LXR5cGUvaW5mb2dyYXBoaWMiLCJjYWFzOmNvbnRlbnQtdHlwZS9lYm9vayIsImNhYXM6Y29udGVudC10eXBlL2d1aWRlIiwiY2Fhczpjb250ZW50LXR5cGUvd2hpdGUtcGFwZXIiLCJjYWFzOmNvbnRlbnQtdHlwZS9ldmVudC1zZXNzaW9uIiwiY2Fhczpjb250ZW50LXR5cGUvZGVtbyIsImNhYXM6Y29udGVudC10eXBlL2NvdXJzZSJdfV0sImJvb2ttYXJrSWNvblNlbGVjdCI6IiIsImJvb2ttYXJrSWNvblVuc2VsZWN0IjoiIiwiY2FyZFN0eWxlIjoiMToyIiwiY29sbGVjdGlvbkJ0blN0eWxlIjoicHJpbWFyeSIsImNvbnRhaW5lciI6IjEyMDBNYXhXaWR0aCIsImNvdW50cnkiOiJjYWFzOmNvdW50cnkvdXMiLCJjb250ZW50VHlwZVRhZ3MiOlsiY2Fhczpjb250ZW50LXR5cGUvcmVwb3J0IiwiY2Fhczpjb250ZW50LXR5cGUvdmlkZW8iLCJjYWFzOmNvbnRlbnQtdHlwZS93ZWJpbmFyIiwiY2Fhczpjb250ZW50LXR5cGUvaW5mb2dyYXBoaWMiLCJjYWFzOmNvbnRlbnQtdHlwZS9lYm9vayIsImNhYXM6Y29udGVudC10eXBlL2RlbW8iLCJjYWFzOmNvbnRlbnQtdHlwZS9ndWlkZSIsImNhYXM6Y29udGVudC10eXBlL2NvdXJzZSIsImNhYXM6Y29udGVudC10eXBlL2FydGljbGUiLCJjYWFzOmNvbnRlbnQtdHlwZS93aGl0ZS1wYXBlciIsImNhYXM6Y29udGVudC10eXBlL2V2ZW50LXNlc3Npb24iXSwiZGlzYWJsZUJhbm5lcnMiOmZhbHNlLCJkcmFmdERiIjpmYWxzZSwiZW52aXJvbm1lbnQiOiIiLCJlbmRwb2ludCI6Ind3dy5hZG9iZS5jb20vY2hpbWVyYS1hcGkvY29sbGVjdGlvbiIsImV4Y2x1ZGVUYWdzIjpbImNhYXM6ZXZlbnRzL21heCIsImNhYXM6Y29udGVudC10eXBlL2N1c3RvbWVyLXN0b3J5Il0sImV4Y2x1ZGVkQ2FyZHMiOlt7ImNvbnRlbnRJZCI6IjY0YzdhZDJlLTdiZjEtNWU3Zi04NjE5LWM2NmYyNzM3MTQ0NSJ9LHsiY29udGVudElkIjoiMGRmZDIyYjQtNTZkMS01Y2U1LWFkNmUtYjU4YzVlZjYwOGQ1In0seyJjb250ZW50SWQiOiJkNjliYzQwMy01OGQ1LTVmY2YtOWZhZC00ZDAwNmY2Yjk2ZDkifSx7ImNvbnRlbnRJZCI6ImJmYzMyZjA2LTZlNmEtNWY5OS1iMjUyLTQ4YzE0MTBiZTVhMyJ9LHsiY29udGVudElkIjoiYzgwZDllZGQtOGM2ZC01ZmMyLTk3ZjEtNDQ1ZjUzZTQ5MTg4In1dLCJmYWxsYmFja0VuZHBvaW50IjoiIiwiZmVhdHVyZWRDYXJkcyI6W10sImd1dHRlciI6IjR4IiwiaW5jbHVkZVRhZ3MiOltdLCJsYW5ndWFnZSI6ImNhYXM6bGFuZ3VhZ2UvZW4iLCJsYXlvdXRUeXBlIjoiM3VwIiwibG9hZE1vcmVCdG5TdHlsZSI6Im92ZXItYmFja2dyb3VuZCIsIm9ubHlTaG93Qm9va21hcmtlZENhcmRzIjpmYWxzZSwib3JMb2dpY1RhZ3MiOltdLCJwYWdpbmF0aW9uQW5pbWF0aW9uU3R5bGUiOiJwYWdlZCIsInBhZ2luYXRpb25FbmFibGVkIjp0cnVlLCJwYWdpbmF0aW9uUXVhbnRpdHlTaG93biI6dHJ1ZSwicGFnaW5hdGlvblVzZVRoZW1lMyI6dHJ1ZSwicGFnaW5hdGlvblR5cGUiOiJwYWdpbmF0b3IiLCJwbGFjZWhvbGRlclVybCI6Imh0dHBzOi8vbWFpbi0tYmFjb20tLWFkb2JlY29tLmhseC5wYWdlL2llL2NhYXMvbWFwcGluZ3MiLCJyZXN1bHRzUGVyUGFnZSI6IjkiLCJzZWFyY2hGaWVsZHMiOlsib3ZlcmxheXMuYmFubmVyLmRlc2NyaXB0aW9uIiwiY29udGVudEFyZWEuZGVzY3JpcHRpb24iLCJjb250ZW50QXJlYS5kZXRhaWxUZXh0IiwiY29udGVudEFyZWEudGl0bGUiLCJvdmVybGF5cy5sYWJlbC5kZXNjcmlwdGlvbiJdLCJzZXRDYXJkQm9yZGVycyI6dHJ1ZSwic2hvd0Jvb2ttYXJrc0ZpbHRlciI6ZmFsc2UsInNob3dCb29rbWFya3NPbkNhcmRzIjpmYWxzZSwic2hvd0ZpbHRlcnMiOnRydWUsInNob3dTZWFyY2giOnRydWUsInNob3dUb3RhbFJlc3VsdHMiOnRydWUsInNvcnREZWZhdWx0IjoiZGF0ZURlc2MiLCJzb3J0RW5hYmxlUG9wdXAiOnRydWUsInNvcnRFbmFibGVSYW5kb21TYW1wbGluZyI6ZmFsc2UsInNvcnRSZXNlcnZvaXJTYW1wbGUiOjMsInNvcnRSZXNlcnZvaXJQb29sIjoxMDAwLCJzb3VyY2UiOlsiYmFjb20iXSwidGFnc1VybCI6Ind3dy5hZG9iZS5jb20vY2hpbWVyYS1hcGkvdGFncyIsInRhcmdldEFjdGl2aXR5IjoiIiwidGFyZ2V0RW5hYmxlZCI6ZmFsc2UsInRoZW1lIjoibGlnaHRlc3QiLCJ0b3RhbENhcmRzVG9TaG93IjoiMTUwMCIsInVzZUxpZ2h0VGV4dCI6ZmFsc2UsInVzZU92ZXJsYXlMaW5rcyI6ZmFsc2UsInVzZXJJbmZvIjpbXSwiZmlsdGVyRXZlbnQiOiIiLCJmaWx0ZXJMb2NhdGlvbiI6ImxlZnQiLCJmaWx0ZXJMb2dpYyI6Im9yIiwiZmlsdGVycyI6W3siZmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMiXSwib3BlbmVkT25Mb2FkIjoiIiwiaWNvbiI6IiIsImV4Y2x1ZGVUYWdzIjpbImNhYXM6cHJvZHVjdHMvbWFnZW50by1jb21tZXJjZSJdfSx7ImZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZSJdLCJvcGVuZWRPbkxvYWQiOiIiLCJpY29uIjoiIiwiZXhjbHVkZVRhZ3MiOlsiY2Fhczpjb250ZW50LXR5cGUvY3VzdG9tZXItc3RvcnkiXX0seyJmaWx0ZXJUYWciOlsiY2Fhczp0b3BpYyJdLCJvcGVuZWRPbkxvYWQiOiIiLCJpY29uIjoiIiwiZXhjbHVkZVRhZ3MiOlsiY2Fhczp0b3BpYy9lbnRlcnByaXNlLXdvcmstbWFuYWdlbWVudCJdfSx7ImZpbHRlclRhZyI6WyJjYWFzOmluZHVzdHJ5Il0sIm9wZW5lZE9uTG9hZCI6IiIsImljb24iOiIiLCJleGNsdWRlVGFncyI6W119XSwiZmlsdGVyc1Nob3dFbXB0eSI6ZmFsc2UsImNvbGxlY3Rpb25TaXplIjoiIiwic29ydEZlYXR1cmVkIjpmYWxzZSwic29ydERhdGVBc2MiOmZhbHNlLCJzb3J0RGF0ZURlc2MiOnRydWUsInNvcnRFdmVudFNvcnQiOmZhbHNlLCJzb3J0VGl0bGVBc2MiOmZhbHNlLCJzb3J0VGl0bGVEZXNjIjpmYWxzZSwic29ydFJhbmRvbSI6ZmFsc2UsImNvbGxlY3Rpb25OYW1lIjoiIiwiZG9Ob3RMYXp5TG9hZCI6dHJ1ZSwiYWRkaXRpb25hbFJlcXVlc3RQYXJhbXMiOltdLCJjYXJkVGl0bGVBY2Nlc3NpYmlsaXR5TGV2ZWwiOjYsImN1c3RvbUNhcmQiOiIiLCJjdGFBY3Rpb24iOiJfYmxhbmsiLCJ0aXRsZUhlYWRpbmdMZXZlbCI6ImgzIiwiY29sbGVjdGlvbkJ1dHRvblN0eWxlIjoicHJpbWFyeSIsInNvcnREYXRlTW9kaWZpZWQiOmZhbHNlLCJhdXRvQ291bnRyeUxhbmciOnRydWUsImNvbGxlY3Rpb25UaXRsZSI6IiIsInNvcnRNb2RpZmllZEFzYyI6ZmFsc2UsInNvcnRNb2RpZmllZERlc2MiOmZhbHNlLCJzaG93SWRzIjp0cnVlLCJmaWx0ZXJCdWlsZFBhbmVsIjoiY3VzdG9tIiwiZmlsdGVyc0N1c3RvbSI6W3siZmlsdGVyVGFnIjoiIiwib3BlbmVkT25Mb2FkIjoiIiwiaWNvbiI6IiIsImV4Y2x1ZGVUYWdzIjoiIiwiZ3JvdXAiOiJQcm9kdWN0cyIsImZpbHRlcnNDdXN0b21JdGVtcyI6W3siZmlsdGVyc0N1c3RvbUxhYmVsIjoiQWRvYmUgRXhwZXJpZW5jZSBQbGF0Zm9ybSIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL2Fkb2JlLWV4cGVyaWVuY2UtcGxhdGZvcm0iXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJBZHZlcnRpc2luZyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL2Fkb2JlLWFkdmVydGlzaW5nLWNsb3VkIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQW5hbHl0aWNzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvYWRvYmUtYW5hbHl0aWNzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQXVkaWVuY2UgTWFuYWdlciIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL2Fkb2JlLWF1ZGllbmNlLW1hbmFnZXIiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJDYW1wYWlnbiIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL2Fkb2JlLWNhbXBhaWduIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ29tbWVyY2UiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9hZG9iZS1jb21tZXJjZSJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkN1c3RvbWVyIEpvdXJuZXkgQW5hbHl0aWNzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvY3VzdG9tZXItam91cm5leS1hbmFseXRpY3MiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJFeHBlcmllbmNlIE1hbmFnZXIiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9hZG9iZS1leHBlcmllbmNlLW1hbmFnZXIiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJFeHBlcmllbmNlIE1hbmFnZXIgQXNzZXRzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvYWRvYmUtZXhwZXJpZW5jZS1tYW5hZ2VyLWFzc2V0cyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkV4cGVyaWVuY2UgTWFuYWdlciBGb3JtcyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL2Fkb2JlLWV4cGVyaWVuY2UtbWFuYWdlci1mb3JtcyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkV4cGVyaWVuY2UgTWFuYWdlciBHdWlkZXMiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9leHBlcmllbmNlLW1hbmFnZXItZ3VpZGVzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRXhwZXJpZW5jZSBNYW5hZ2VyIFNpdGVzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvYWRvYmUtZXhwZXJpZW5jZS1tYW5hZ2VyLXNpdGVzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiSm91cm5leSBPcHRpbWl6ZXIiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9hZG9iZS1qb3VybmV5LW9wdGltaXplciJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkxlYXJuaW5nIE1hbmFnZXIiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9sZWFybmluZy1tYW5hZ2VyIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiTWFya2V0byBFbmdhZ2UiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9tYXJrZXRvLWVuZ2FnZS1iaXppYmxlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiUHJvZHVjdCBBbmFseXRpY3MiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9hZG9iZS1wcm9kdWN0LWFuYWx5dGljcyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlByb2Zlc3Npb25hbCBTZXJ2aWNlcyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnByb2R1Y3RzL3Byb2Zlc3Npb25hbC1zZXJ2aWNlcyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlJlYWwtVGltZSBDRFAiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpwcm9kdWN0cy9hZG9iZS1yZWFsLXRpbWUtY2RwIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiVGFyZ2V0IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvYWRvYmUtdGFyZ2V0Il19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiV29ya2Zyb250IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6cHJvZHVjdHMvd29ya2Zyb250Il19XX0seyJmaWx0ZXJUYWciOiIiLCJvcGVuZWRPbkxvYWQiOiIiLCJpY29uIjoiIiwiZXhjbHVkZVRhZ3MiOiIiLCJncm91cCI6IkNvbnRlbnQgVHlwZSIsImZpbHRlcnNDdXN0b21JdGVtcyI6W3siZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ29udGVudCBUeXBlIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6Y29udGVudC10eXBlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQXJ0aWNsZSIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZS9hcnRpY2xlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRGVtbyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZS9kZW1vIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiZUJvb2siLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpjb250ZW50LXR5cGUvZWJvb2siXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJFdmVudCBTZXNzaW9uIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6Y29udGVudC10eXBlL2V2ZW50LXNlc3Npb24iXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJHdWlkZSIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZS9ndWlkZSJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkluZm9ncmFwaGljIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6Y29udGVudC10eXBlL2luZm9ncmFwaGljIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiUHJvZHVjdCBUb3VyIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6Y29udGVudC10eXBlL3Byb2R1Y3QtdG91ciJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlJlcG9ydCIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZS9yZXBvcnQiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJWaWRlbyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmNvbnRlbnQtdHlwZS92aWRlbyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IldlYmluYXIiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpjb250ZW50LXR5cGUvd2ViaW5hciJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IldoaXRlcGFwZXIiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczpjb250ZW50LXR5cGUvd2hpdGUtcGFwZXIiXX1dfSx7Imdyb3VwIjoiVG9waWMiLCJmaWx0ZXJzQ3VzdG9tSXRlbXMiOlt7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlRvcGljIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJBZHZlcnRpc2luZyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2FkdmVydGlzaW5nIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQW5hbHl0aWNzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvYW5hbHl0aWNzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQXJ0aWZpY2lhbCBJbnRlbGxpZ2VuY2UiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczp0b3BpYy9hcnRpZmljaWFsLWludGVsbGlnZW5jZSJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkJ1c2luZXNzIGltcGFjdCIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2J1c2luZXNzLWltcGFjdCJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkNhbXBhaWduIG1hbmFnZW1lbnQiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczp0b3BpYy9jYW1wYWlnbi1tYW5hZ2VtZW50Il19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ2FtcGFpZ24gb3JjaGVzdHJhdGlvbiIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2NhbXBhaWduLW9yY2hlc3RyYXRpb24iXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJDb21tZXJjZSIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2NvbW1lcmNlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ29udGVudCBNYW5hZ2VtZW50IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvY29udGVudC1tYW5hZ2VtZW50Il19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ3VzdG9tZXIgSW50ZWxsaWdlbmNlIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvY3VzdG9tZXItaW50ZWxsaWdlbmNlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiQ3VzdG9tZXIgSm91cm5leSBNYW5hZ2VtZW50IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvY3VzdG9tZXItam91cm5leS1tYW5hZ2VtZW50Il19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRGF0YSAmIEluc2lnaHRzIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvZGF0YS0mLWluc2lnaHRzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRGF0YSBGb3VuZGF0aW9uIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvZGF0YS1mb3VuZGF0aW9uIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRGF0YSBNYW5hZ2VtZW50IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvZGF0YS1tYW5hZ2VtZW50Il19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiRGVtYW5kIE1hcmtldGluZyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2RlbWFuZC1tYXJrZXRpbmciXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJEaWdpdGFsIEFzc2V0IE1hbmFnZW1lbnQiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczp0b3BpYy9kaWdpdGFsLWFzc2V0LW1hbmFnZW1lbnQiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJEaWdpdGFsIEZvdW5kYXRpb24iLCJjdXN0b21GaWx0ZXJUYWciOlsiY2Fhczp0b3BpYy9kaWdpdGFsLWZvdW5kYXRpb24iXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJEaWdpdGFsIFRyZW5kcyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2RpZ2l0YWwtdHJlbmRzIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiR2VuZXJhdGl2ZSBBSSIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL2dlbmVyYXRpdmUtYWkiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJNYXJrZXRpbmcgQXV0b21hdGlvbiIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOnRvcGljL21hcmtldGluZy1hdXRvbWF0aW9uIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiUGVyc29uYWxpemF0aW9uIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvcGVyc29uYWxpemF0aW9uIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiV29yayBNYW5hZ2VtZW50IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6dG9waWMvd29yay1tYW5hZ2VtZW50Il19XSwib3BlbmVkT25Mb2FkIjoiIn0seyJncm91cCI6IkluZHVzdHJ5IiwiZmlsdGVyc0N1c3RvbUl0ZW1zIjpbeyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJBdXRvbW90aXZlICYgTW9iaWxpdHkiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2FhczppbmR1c3RyeS9hdXRvbW90aXZlLSYtbW9iaWxpdHkiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJGaW5hbmNpYWwgU2VydmljZXMiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2FhczppbmR1c3RyeS9maW5hbmNpYWwtc2VydmljZXMiXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJHb3Zlcm5tZW50IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6aW5kdXN0cnkvZ292ZXJubWVudCJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IkhlYWx0aGNhcmUiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2FhczppbmR1c3RyeS9oZWFsdGhjYXJlIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiSGlnaCBUZWNoIiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6aW5kdXN0cnkvaGlnaC10ZWNoIl19LHsiZmlsdGVyc0N1c3RvbUxhYmVsIjoiTWFudWZhY3R1cmluZyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmluZHVzdHJ5L21hbnVmYWN0dXJpbmciXX0seyJmaWx0ZXJzQ3VzdG9tTGFiZWwiOiJNZWRpYSAmIEVudGVydGFpbm1lbnQiLCJjdXN0b21GaWx0ZXJUYWciOlsiY2FhczppbmR1c3RyeS9tZWRpYS1hbmQtZW50ZXJ0YWlubWVudCJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlJldGFpbCIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmluZHVzdHJ5L3JldGFpbCJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlRlbGVjb21tdW5pY2F0aW9ucyIsImN1c3RvbUZpbHRlclRhZyI6WyJjYWFzOmluZHVzdHJ5L3RlbGVjb21tdW5pY2F0aW9ucyJdfSx7ImZpbHRlcnNDdXN0b21MYWJlbCI6IlRyYXZlbCAmIEhvc3BpdGFsaXR5IiwiY3VzdG9tRmlsdGVyVGFnIjpbImNhYXM6aW5kdXN0cnkvdHJhdmVsLWFuZC1ob3NwaXRhbGl0eSJdfV0sIm9wZW5lZE9uTG9hZCI6IiJ9XSwiaGlkZUN0YUlkcyI6W10sImRldGFpbHNUZXh0T3B0aW9uIjoiZGVmYXVsdCJ9) (See filters panel)
- After: [https://ui-controls-update--milo--adobecom.hlx.page/?martech=off](https://ui-controls-update--milo--adobecom.hlx.page/tools/caas?martech=off)

### CODE REVIEW

File: libs/blocks/caas/caas.css [hunk 1]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
 }
 /* End Dexter Modal */
+
+/* CaaS Overrrides */
+.tool-content {
+    grid-template-columns: 340px 20px 1fr;
+    gap: 6px;
+}
+
+dt.title span {
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:384]: Isn't `dt` a bit generic, should it be scoped to a certain parent?

File: libs/blocks/caas/caas.css [hunk 2]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+  border-bottom: 0;
+  padding: 0;
+  margin: 14px 0 0;
+}
+
+.multifield > .multifield-header h3 {
+  font-size: 18px;
+  font-weight: 700;
+  margin: .5rem;
+  margin: 2px 6px;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:403]: Can't these two be merged?

File: libs/blocks/caas/caas.css [hunk 3]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+dd.content .field {
+  margin: 8px 0;
+}
+
+.accordion-item {
+  margin: 1px 0;
+}
+
+.accordion > .accordion-item {
+    margin: 1px 0;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:427]: Extra indentation; rule is identical t the previous, they can be grouped together

File: libs/blocks/caas/caas.css [hunk 4]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+}
+
+.accordion > .accordion-item .title {
+    cursor: pointer;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:431]: Extra indentation

File: libs/blocks/caas/caas.css [hunk 5]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+  max-width: 220px;
+}
+
+input[type=checkbox] {
+    align-self: baseline;
+    margin-top: 6px;
+}
+
+button.collapse-panel {
+    /* transform: rotate(90deg); */
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:492]: Is this still needed, since it's commented out?

File: libs/blocks/caas/caas.css [hunk 6]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+    border: none;
+    font-size: 18px;
+    width: 26px;
+    height: 32px;
+    cursor: pointer;
+}
+button.collapse-panel:hover {
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:499]: New line needed before

File: libs/blocks/caas/caas.css [hunk 7]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+  background:#666;
+  color: #eee;
+}
+
+.panel-collapsed .tool-content {
+    grid-template-columns: 20px 20px 1fr;
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:505]: Extra indentation, there are other occurrences, all should be addressed

File: libs/blocks/caas/caas.css [hunk 8]
```diff
@@ -374,3 +374,156 @@ main > .section > .content .consonant-Wrapper--1600MaxWidth .consonant-Wrapper-i
...
+.panel-collapsed dt.title span {
+    padding: 0 3px
+}
+
+.panel-collapsed button {
+  cursor:pointer;
+}
+/* .panel-collapsed button.collapse-panel {
+    transform: rotate(-90deg);
+} */
```
Comment (overmyheadandbody) [inline libs/blocks/caas/caas.css:524]: Can this be removed?

**Review by overmyheadandbody — COMMENTED** [review]
ESLint issues should be addressed to make life a bit easier for reviewers. Ideally, you should set up ESLint within your IDE to spot these before opening the PR.

**Review by sanrai — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

**Comment by honstar** [comment]
There are still some linting issues, please check, thx.

**Comment by cmiqueo** [comment]
> There are still some linting issues, please check, thx.

Looks like the lint error is unrelated to this PR changes:
```
Check failure on line 71 in libs/ui/controls/formControls.js

GitHub
Actions
/ Running eslint

[eslint] reported by reviewdog 🐶
Unary operator '++' used.
```

**Pull request merged**